packages/webpack-config/webpack/loaders/createBabelLoaderAsync.js

```diff
@@ -75,18 +67,18 @@ module.exports = async function({
 
     include(inputPath) {
       for (const possibleModule of modules) {
-        if (inputPath.includes(possibleModule)) {
+        if (inputPath.match(possibleModule)) {
```

Comment:
`modules` has some strings and some regexes in it. it's probably going to be confusing that strings are converted to regexes automatically. I think the config will need to document more clearly how the `include` option is supposed to work:

```
module.exports = async function({
  /**
   * The webpack mode: `"production" | "development"`
   */
  mode,
  babelProjectRoot,
  include = [],
  verbose,
  ...options
} = {}) {
  const ensuredProjectRoot = await ensureRootAsync(babelProjectRoot);
  const modules = [...includeModulesThatContainPaths, ...include];
```